### PR TITLE
Video fix

### LINKED ISSUE #10: Review quality of video/mov file in readme
**I would like to see this change implemented:**

- See if there is a way to improve the quality of the video/.mov file in the readme documentation

**Additional context or further questions:**

- RS had a look previously but couldn't find anything; GH requires downsizing video which lowers the quality

### LINKED ISSUE #12: Should we create code-for-gold?
Following the structure of [code-for-aurum](https://github.com/aim-rsf/cprd-data-wrangle//tree/main/code-for-aurum), create code-for-gold based on the medium fidelity synthetic gold dataset? See if there is a need for this, and consider improving code-for-aurum first.

### PR DESCRIPTION
**Pull request by Rainiefantasy**
Addresses #10 

Quality of .mov file was lowered too much, was a bit blurry, and link had broken so needed a fix.

## Proposed Changes

1. Converted .mov file into gif using ffmpeg
2. Added gif to the home folder
3. added gif link ref in the readme

## Checklist before review:

- [x] Are you happy with the gif file being in the main folder? if not can move it
- [x] check if it works for you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ We include information on [CPRD's Code Browser tool](cprd-code-browser.md) and h
 
 The [code-for-aurum](code-for-aurum) folder uses `Python` and `postgreSQL` to create a pre-processing workflow for CPRD Aurum data which includes a conversion of data file format for compatibility, and then reading the data into tables in a relational database. Workbooks have been created to familiarise a user with the CPRD Aurum tables, including how they link together and how to build a sample cohort:
 
-https://github.com/user-attachments/assets/9a636d4c-8170-4145-b6fc-60ac0f4c16d1
+![landing-page-demo-gif](https://github.com/aim-rsf/cprd-data-wrangle/blob/video-fix/output.gif?raw=true)
```
Comment (RayStick) [inline README.md:43]:
```suggestion
![landing-page-demo-gif](https://github.com/aim-rsf/cprd-data-wrangle/blob/video-fix/notebook_demo.gif?raw=true)
```

Comment (RayStick) [reply README.md:43]: Could we rename the .gif file to something clearer like 'notebook_demo.gif'?
If that works for you, please change the name of the gif file :)

**Comment by RayStick** [comment]
This looks great! Such an improvement. Thanks for working on it

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ We include information on [CPRD's Code Browser tool](cprd-code-browser.md) and h
 
 The [code-for-aurum](code-for-aurum) folder uses `Python` and `postgreSQL` to create a pre-processing workflow for CPRD Aurum data which includes a conversion of data file format for compatibility, and then reading the data into tables in a relational database. Workbooks have been created to familiarise a user with the CPRD Aurum tables, including how they link together and how to build a sample cohort:
```
Comment (RayStick) [inline README.md:41]:
```suggestion
The [code-for-aurum](code-for-aurum) folder uses `Python` and `postgreSQL` to create a pre-processing workflow for CPRD Aurum data which includes a conversion of data file format for compatibility, and then reading the data into tables in a relational database. Workbooks have been created to familiarise a user with the CPRD Aurum tables, including how they link together and how to build a sample cohort. See a preview below:
```

**Review by RayStick — APPROVED** [review]

**Comment by RayStick** [comment]
This looks great! Such an improvement. Thanks for working on it

Two suggestions for you to implement (via 'commit suggestion' and also file name change) - after that, happy for you to merge it!

File: README.md (hunk 1 above)
Comment (Rainiefantasy) [reply README.md]: Sorry I changed it manually - I could've just commited XD ah well

**Comment by Rainiefantasy** [comment]
I've committed your suggestion for readme changes - I realise I preemptively changed the filename after reading your suggestion when I could've committed your suggestion instead sorry!

**Pull request merged**